pythran/tests/test_cases.py

```diff
@@ -16,6 +17,10 @@ class TestCases(TestFromDir):
 
 TestCases.populate(TestCases)
 
+if numpy_version() >= (1, 20, 0):
```

Comment:
There would be an easier way without having to worry about rolling your own `numpy_version`:
```suggestion
if LooseVersion(numpy.__version__) >= "1.20":
```

together with an import: `from distutils.version import LooseVersion`